src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const matches = document.querySelectorAll('span.population');
+let total = 0;
+
+for (const element of matches) {
+  const population = +element.textContent.replaceAll(',', '');
+
+  total += isNaN(population) ? 0 : +population;
+}
+
+document.querySelector('span.total-population').innerHTML =
+  Intl.NumberFormat('en-US').format(total);
+
+document.querySelector('span.average-population').innerHTML = Intl.NumberFormat(
+  'en-US',
+).format(total / matches.length);
```

Comment:
There is a potential issue when calculating the average population. If there are no elements with the class 'population', `matches.length` will be zero, leading to a division by zero error. Consider adding a check to ensure `matches.length` is not zero before performing the division.